src/keycloak/chart/templates/uds-package.yaml

```diff
@@ -73,6 +93,28 @@ spec:
         {{- end }}
       {{- end }}
 
+      - description: "Keycloak to Waypoint"
+        direction: Ingress
+        remoteGenerated: IntraNamespace
+        ports:
+          - 15008
+          - 8080
+      - description: "Waypoint to Keycloak"
+        direction: Egress
+        remoteGenerated: IntraNamespace
+        ports:
+          - 15008
+          - 8080
```

Comment:
Similar comment on the ports here - we shouldn't need 15008 since the operator will add that ([ref](https://github.com/defenseunicorns/uds-core/blob/main/src/pepr/operator/controllers/network/policies.ts#L154-L177)).

Reply:
Fixed. Thanks for spotting this.